src/plot/src/main/python/plot.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
Plotting should go into its own sub-module.. please file an issue if you are not able to refactor for this check-in.

Reply:
This is in its own submodule. If a python class is not explicitly imported in the __init__.py its addressable by the filename aka mmlspark.plot